### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by didymus707**
Project completed

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+.phone-list li a img {
+  width: 336px; 
+  height: 200px;
+}
+/* gallery style ends */
+
+/* fotter styles */
+.areas {
+  /* height: 10vh; */
```
Comment (Redvanisation) [inline css/style.css]: Please delete commented code

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+.cta {
+  height: 31px;
+  overflow: hidden;
+}
+/* wrapper styles ends */
+
+/* gallery style starts */
+.gallery {
+  /* height: 29vh; */
```
Comment (Redvanisation) [inline css/style.css]: Commented code should be deleted

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  🛑

### Hello @didymus707   🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. Please center the 4 bottom elements to the center of the page.

2. Set up Stickler to lint the CSS on your project, find out more on how to do that in [here](https://github.com/microverseinc/linters-config/tree/master/css)

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

File: css/style.css (hunk 2 above)
Comment (didymus707) [reply css/style.css]: changes made

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```
Comment (suretrust) [inline .editorconfig]: This should be added to `gitignore`, let's delete the file from the repo.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+  height: 31px;
+  overflow: hidden;
+}
+/* wrapper styles ends */
+
+/* gallery style starts */
+.phone-list {
+  display: flex;
+  justify-content: center;
+}
```
Comment (suretrust) [inline css/style.css:142]: The total width of these images should be the same as the width of the background image above it.

width

File: index.html [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years_2x.jpg" alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+        </a>
+      </li>
+      <li>
+        <a href="#" class="i-4">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility_2x.jpg" alt="Supplier Responsibility The latest results of our worldwide efforts.">
+        </a>
+      </li>
+    </ul>
+  </div>
```
Comment (suretrust) [inline index.html]: You can extend the `<main>` up to here since everything above footer is still main content.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @didymus707,

### Great job thus far, your `readME` is lit! Please see my comments on your code.

Also, let's set up the linter properly, it should display as shown below when properly setup.
linter2

**Kindly submit for another code review once these issues have been fixed.** _Happy coding!_ :tada:

File: index.html (hunk 5 above)
Comment (didymus707) [reply index.html]: All changes made

File: css/style.css (hunk 4 above)
Comment (didymus707) [reply css/style.css:142]: Changes made

File: .editorconfig (hunk 3 above)
Comment (didymus707) [reply .editorconfig]: Changes made

**Review by Redvanisation — APPROVED** [review]
# Project approved ✔️ 

### Hello @didymus707  🌟 

### _Nice work on this project, it seems that you have met the requirements so I am approving your project. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

**Pull request merged**